amr-wind/wind_energy/actuator/FLLC.H

```diff
@@ -33,6 +33,13 @@ struct FLLCData
     bool different_sizes;
     RealList span_distance_vel;
     RealList span_distance_force;
+
+    // non-uniform variables
+    bool nonu{true};    // non-uniform flag
```

Comment:
If you need two comments to explain what a boolean variable is then the name is not good. Abbreviating is not doing anyone any favors here. I would just call in `nonuniform` and delete the comments.

Reply:
Agreed, fixed in latest version.